app/views/locations/_form.html.erb

```diff
@@ -50,17 +50,21 @@
           </span>
         </li>
         <li>
-          <%= f.check_box :coordinateable, data: {toggle: "coordinates"} %>
+          <%= f.check_box :coordinateable, data: {toggle: "coordinates"}, disabled: !locals[:can_toggle_coordinates] %>
           <%= f.label :coordinateable, "Has Co-ordinates" %>
         </li>
         <div class="coordinate-options" data-behavior="coordinates">
           <li>
-            <%= f.label :rows %>
-            <%= f.text_field :rows %>
-          </li>
-           <li>
-            <%= f.label :columns %>
-            <%= f.text_field :columns %>
+            <div class="table">
+              <div class="table-row">
+                <div class="table-head"><%= f.label :rows %></div>
+                <div class="table-head"><%= f.label :columns %></div>
+              </div>
+              <div class="table-row">
+                <div class="table-cell"><%= f.number_field :rows, min: 0 %></div>
+                <div class="table-cell"><%= f.number_field :columns, min: 0 %></div>
+              </div>
+            </div>
```

Comment:
Does this change the styling? Can I have a screenshot?

Reply:
![LabwhereCoordinates](https://user-images.githubusercontent.com/58506299/96839156-b14ce080-1440-11eb-9e9b-ee2f34017bbe.png)

Reply:
It does change the styling. Happy to revert or try an alternative styling if you'd prefer

Reply:
In fact I think I'll revert back to the two list entries

Reply:
![LabwhereCoordinatesNew](https://user-images.githubusercontent.com/58506299/96842452-f2df8a80-1444-11eb-8adb-24a6c79ed22a.png)

Reply:
Not bothered either way, was just interested to see the result. Thanks :)